### PR TITLE
improve: refactor partial command line tool.

### PR DESCRIPTION
**Pull request by tokers**
### What this PR does / why we need it:

This PR moves some environmental variables to `apisix/cli/environ.lua`,
and some auxiliary codes to `apisix/cli/util.lua`, which reduces the size
of `bin/apisix`.

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [x] Is this PR backward compatible?

### CODE REVIEW

**Review by membphis — COMMENTED** [review]
`apisix/cli/environ.lua` to `apisix/cli/env.lua`

how about this?

**Comment by tokers** [comment]
> `apisix/cli/environ.lua` to `apisix/cli/env.lua`
> 
> how about this?

Agreed, `env.lua` is more concise, actually `environ` is a function in libc and i just copy this name.

**Review by membphis — APPROVED** [review]
very nice PR

File: apisix/cli/util.lua [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+-- it is recommended to use the `trim` function to handle the return value.
+function _M.execute_cmd(cmd)
+    local t, err = popen(cmd)
+    if not t then
+        return nil, "failed to execute command: "
+                    .. cmd .. ", error info: " .. err
+    end
+
+    local data, err = t:read("*all")
+    if err ~= nil then
```
Comment (spacewander) [inline apisix/cli/util.lua:33]: Should always close `t` even error happened.

File: bin/apisix [hunk 2]
```diff
@@ -346,8 +285,8 @@ end
 
 local checked_admin_key = false
 local function init()
-    if is_root_path then
-        print('Warning! Running apisix under /root is only suitable for development environments'
+    if env.is_root_path then
+        print('Warning! Running apisix under /root is only suitable for development envments'
```
Comment (spacewander) [inline bin/apisix:289]: Why use `envments`?

**Review by spacewander — CHANGES_REQUESTED** [review]

File: bin/apisix (hunk 2 above)
Comment (tokers) [reply bin/apisix:289]: Oops, it's the result of full replace from "environ" to "env".

**Review by spacewander — APPROVED** [review]

**Pull request merged**